src/app/FakeLib/CscHelper.fs

```diff
@@ -78,11 +78,21 @@ let cscExe toolPath (srcFiles : string list) (opts : string list) : int =
 ///                                Target = ...
 ///                                ... })
 let csc (setParams : CscParams -> CscParams) (inputFiles : string list) : int =
-    let inputFiles = inputFiles |> Seq.toList
+    // Helper to quote a path with spaces in it, if not already quoted.  See https://github.com/fsharp/FAKE/issues/992
+    let ensureQuotedPath (path : string) =
+        // Sensitive to being backwards compatible with people that are using
+        // Csc AND quoting their paths.  Only quote if space in path and quotes not detected.
+        // MAYBE this should go in the FileSystemHelper module?
+        if path.Contains(" ") then
```

Comment:
Thinking this should probably be trimmed.  Currently we could double quote a path that is already quoted, but book ended with whitespace.